modules/openxr/openxr_api_extension.h

```diff
@@ -82,6 +82,23 @@ class OpenXRAPIExtension : public RefCounted {
 	void register_composition_layer_provider(OpenXRExtensionWrapperExtension *p_extension);
 	void unregister_composition_layer_provider(OpenXRExtensionWrapperExtension *p_extension);
 
+	void register_projection_views_extension(OpenXRExtensionWrapperExtension *p_extension);
+	void unregister_projection_views_extension(OpenXRExtensionWrapperExtension *p_extension);
+
+	double get_render_state_z_near();
+	double get_render_state_z_far();
+
+	void set_motion_vector_texture(RID p_render_target);
+	void set_motion_vector_depth_texture(RID p_render_target);
+	void set_motion_vector_target_size(Size2i p_target_size);
+
+	uint64_t openxr_swap_chain_info_create(XrSwapchainCreateFlags p_create_flags, XrSwapchainUsageFlags p_usage_flags, int64_t p_swapchain_format, uint32_t p_width, uint32_t p_height, uint32_t p_sample_count, uint32_t p_array_size);
+	void openxr_swap_chain_info_free(uint64_t p_swap_chain_info);
+	uint64_t openxr_swap_chain_info_get_swapchain(uint64_t p_swap_chain_info);
+	void openxr_swap_chain_info_acquire(uint64_t p_swap_chain_info);
+	RID openxr_swap_chain_info_get_image(uint64_t p_swap_chain_info);
+	void openxr_swap_chain_info_release(uint64_t p_swap_chain_info);
```

Comment:
I wonder if we should remove the `info` part of these method names? `OpenXRSwapChainInfo` makes sense as the struct name internally, but as a public API, maybe these would be better as `openxr_swap_chain_create()` (without the `info`), etc?

Also, I wonder about `swap_chain` vs `swapchain`: I feel like this is done more commonly as a single word, rather than two. The OpenXR handle is `XrSwapchain` and the [Vulkan spec](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_swapchain.html) always uses "swapchain" as one word. This would also match some existing public functions that use `swapchain` rather than `swap_chain` (like `_set_swapchain_create_info_and_get_next_pointer()` or `_on_main_swapchains_created()`).

So, maybe it should be `openxr_swapchain_create()`, etc?

Reply:
Exposing these feels wrong to me, why are we exposing this to GDExtension? Motion vectors are an internal system in Godot and when motion vectors are required, the `OpenXRAPI` can simply be told to also allocate those buffers where currently the other buffers are being created.

Reply:
There's other OpenXR extensions that require managing swapchains - I think it makes sense to have these.

If we were to have `OpenXRAPI` allocate these swapchains for us, then it would have to know stuff specifically about AppSW. That could be fine in this case, but it would mean we'd have to do something similar for any future OpenXR extensions that need to manage swapchains.

Another option is just having the GDExtension use the OpenXR API directly to manage the swapchains, rather than Godot providing an API to do it? But I think the functionality we get from `OpenXRSwapChainInfo` is nice to have, and standardizes the behavior of OpenXR extensions.